app/models/user.rb

```diff
@@ -107,6 +111,18 @@ def title
     employees.active.first.try(:title).try(:name)
   end
 
+  def employee?
+    employees.active.exists?
+  end
+
+  def student?
+    students.exists?
```

Comment:
`Student` için de aktif olanların seçilmesi gerekmez mi? [Şu satırda](https://github.com/omu/nokul/pull/952/files#diff-211c911748b14934ec69896a8da0efb9R42) aktif olanların öğrenci numaraları alınmış.